business-registry-dashboard/app/components/Table/AffiliatedEntity/index.vue

```diff
@@ -73,7 +89,24 @@ watch(
           padding: 'px-0 py-0'
         },
         td: {
-          base: 'whitespace-normal max-w-96 align-top',
+          base: `
+            /* Default text handling */
+            whitespace-normal
+            align-top
+
+            /* Standard column width constraints */
+            w-48
+            min-w-[192px]
+            max-w-[192px]
+
+            /* Wider first column for business names */
+            [&:first-child]:min-w-[250px]
+            [&:first-child]:max-w-[250px]
+
+            /* Wider third column for type */
+            [&:nth-child(3)]:min-w-[230px]
+            [&:nth-child(3)]:max-w-[230px]
```

Comment:
This looks a bit weird maybe. Here's why I have this:

Before, the user experience was bad. You would select a filter (type for example), and would see the entire columns either shifting left or right. You can try that in the dev link: https://business-registry-dev.web.app/en-CA

From UXA notes, they wanted stable columns
![image](https://github.com/user-attachments/assets/a7da67ed-dcfb-44f1-a865-4e01cce9d5b2)

So what I came up with was this. Basically, the first 3 lines sets a standard column width (no shrinking or growing). The next two parts is basically having a wider width for the name and type. You can try things out in the temp link, it feels really good now.

Reply:
Yes, it generally feels good. Looks like the old Type (and Status) drop-down menus were too narrow for some items.

This still jumps around (in the temp URL) if you select a long type and a long status.